understanding/focus-not-obscured-minimum.html

```diff
@@ -380,90 +275,62 @@ <h4>User openable, persistent disclosures</h4>
+                  <p>外観上でフォーカスを受け取らず、かつオーバーレイに対してインタラクションを制限しない、又はフォーカスを失ったときに自身を閉じない (したがって、フォーカスが背後のコンテンツに抜けるようにする) ダイアログのようなオーバーレイは、他のフォーカス可能な項目を隠すことができるように配置されており、この達成基準で失敗する危険性がある。
                   </p>
                   
                </section>
             </section>
             <section id="benefits">
-               <h2>Benefits</h2>
+               <h2>利点</h2>
                <ul>
                   
-                  <li>Sighted users who rely on a keyboard interface to operate the page will be able to
-                     see the component which gets keyboard focus. Such users include those who rely on
-                     a keyboard or on devices which use the keyboard interface, including speech input,
-                     sip-and-puff software, onscreen keyboards, scanning software, and a variety of assistive
-                     technologies and alternate keyboards.
+                  <li>キーボードインタフェースに依存してページを操作している目の見える利用者は、キーボードフォーカスを取得するコンポーネントを見ることができる。そのような利用者には、キーボード又は、音声入力、呼気・吸気ソフトウェア、オンスクリーンキーボード、スキャンソフトウェア、並びにさまざまな支援技術及び代替キーボードなどのキーボードインタフェースに依存している利用者が含まれる。
                   </li>
                   
-                  <li>People with limited or low vision, who may primarily user a pointer for screen orientation
-                     and repositioning, nonetheless benefit from a visible indication of the current point
-                     of keyboard interaction, especially where magnification reduces the overall viewing
-                     portion of the screen.
+                  <li>視覚に制限のある人々又はロービジョンの人々は、主に画面上の位置取り及び移動にポインタを使用するが、それにもかかわらず、キーボード操作による現在位置の可視的な表示は、特に拡大率によって画面全体の表示視野が狭くなる場合、有益である。
                   </li>
                   
-                  <li>People with attention limitations, short term memory limitations, or limitations in
-                     executive processes benefit by being able to discover where the focus is located.
+                  <li>注意力の制限、短期記憶の制限、又は実行プロセスの制限がある人々は、フォーカスがどこにあるのかを発見できることで恩恵を受ける。
                   </li>
                   
                </ul>
             </section>
             <section id="examples">
-               <h2>Examples</h2>
+               <h2>事例</h2>
                <ul>
                   
-                  <li>A page has a sticky footer (attached to the bottom of the viewport). When tabbing
-                     down the page the focused item is not completely visually obscured by the footer because
-                     content in the viewport scrolls up to always display the item with keyboard focus
-                     using <a href="https://www.w3.org/TR/css-scroll-snap/#propdef-scroll-padding" rel="nofollow">scroll padding</a>.
+                  <li>ページにスティッキーフッター (ビューポートの下部に固定されている) がある。ページをタブで下に移動すると、フォーカスされた項目はフッターによって視覚的に完全に隠れることはない。これは、ビューポート内のコンテンツが上にスクロールして、<a href="https://www.w3.org/TR/css-scroll-snap/#propdef-scroll-padding" rel="nofollow">スクロールパディング</a>を使用してキーボードフォーカスのある項目が常に表示されるためである。
```

Comment:
scroll-padding は CSS のプロパティ名なのでそのままでよいかなと。

```suggestion
                  <li>ページにスティッキーフッター (ビューポートの下部に固定されている) がある。ページをタブで下に移動すると、フォーカスされた項目はフッターによって視覚的に完全に隠れることはない。これは、ビューポート内のコンテンツが上にスクロールして、<a href="https://www.w3.org/TR/css-scroll-snap/#propdef-scroll-padding" rel="nofollow">scroll-padding</a>を使用してキーボードフォーカスのある項目が常に表示されるためである。
```